lesson3/zenameless_hometask3.rb

```diff
@@ -0,0 +1,22 @@
+
+module MyEach
+
+  def my_each
+    retrun self.method(__method__) unless block_given?
```

Comment:
To be honest, I don't understand what do you want to return here.
`Method` object? Why? If you want to create `Enumerator` object when a block is not specified, [read this](http://blog.arkency.com/2014/01/ruby-to-enum-for-enumerator/).